### DISCUSSION TITLE
2024 focus

### DISCUSSION THREAD

**Discussion by CloCkWeRX** [fatfreecrm/fat_free_crm#1249] [Ideas]
@steveyken you seem on a bit of a roll with recent tweaks.

I'm not using fat free CRM in a current professional context at the moment, so I'm not really seeing the pain points or gotchas.

What's on your list of "it'd be great if..."?

I vaguely had an urge to make mobile UX better for the use case of "have sales lunch, add notes".

What's your top 5?

**Comment by steveyken** [comment]
Yes, I'm upgrading my non-proft org from v0.13 to latest!

Next week I'll look at a few issues I've spotted:

* adding a tag to a contact does not load extra custom fields groups inline (it used to)
* ransack advanced search form doesn't seem to convert to select2 boxes anymore
* custom field select boxes should use select2

Once they are done, unless I spot any other deal breakers, I'll likely migrate and put the project down again.

Looking forward, my top 5 would be:

1. Improve mobile friendly UX (as you mentioned)
2. Replace RJS with Stimulus / Turbo (huge undertaking)
3. Improve Wiki documentation (including an example for Azure AD integration - which I've got working locally)
4. Fix bugs in Ransack UI (can't remember what they are)
5. Perhaps add ffcrm_merge to core? (Not sure about this)
